Jenkinsfile

```diff
@@ -100,7 +100,9 @@ pipeline {
                                                 credentials('brave-builds-github-token-for-pr-builder')
                                                 github('brave/devops', 'https')
                                             }
-                                            branch('${params.DEVOPS_BRANCH}')
+
+                                            // FIXME: branch('${params.DEVOPS_BRANCH}')
+                                            branch('wknapik-node-lookup-fix')
```

Comment:
dont forget to revert this! 😄

Reply:
This devops branch is already merged to master, so this can be reverted at any time with no downside.